src/test/java/seedu/staffsnap/ui/ApplicantCardTest.java

```diff
@@ -80,15 +90,45 @@ public void displayApplicantDetails_newApplicant_applicantDetailsDisplayed() {
     }
 
     @Test
-    public void displayApplicantStatus_newApplicant_applicantStatusDisplayed() {
+    public void displayApplicantStatus_applicantStatusDisplayed() {
+        //ApplicantCard should correctly display the status of the applicant
         HBox idLabel = (HBox) stage.getScene().lookup("#details");
         Label status = (Label) idLabel.lookup("#status");
         assertNotNull(status);
         assertEquals("OFFERED", status.getText());
     }
 
     @Test
-    public void displayApplicantScore_newInterview_correctScore() {
+    public void displayApplicantInterviews_correctDataDisplayed() {
+        //ApplicantCard should display the correct title and score for each interview
+        List<Interview> interviewList = BENSON.getInterviews();
+        HBox interviews = (HBox) stage.getScene().lookup("#interviews");
+
+        for (int i = 0; i < interviewList.size(); i++) {
+            Interview interviewToCheck = interviewList.get(i);
+            System.out.println(interviewToCheck.toString());
```

Comment:
Is the print statement intended? Perhaps can remove if it is not necessary 😊

Reply:
thanks for the review Celestine, I've changed it accordingly!